backend/Socket.hs

```diff
@@ -17,42 +18,12 @@ import qualified Compile
 fileChangeApp :: FilePath -> WS.ServerApp
 fileChangeApp watchedFile pendingConnection =
   do  connection <- WS.acceptRequest pendingConnection
+      WS.forkPingThread connection 30
       Notify.withManager $ \notifyManager ->
-        do  _ <- NDevel.treeExtExists notifyManager "." "elm" (sendHotSwap watchedFile connection)
-            keepAlive connection
-
+        do  NDevel.treeExtExists notifyManager "." "elm" (sendHotSwap watchedFile connection)
+            forever $ threadDelay maxBound
```

Comment:
I'm not sure this will correctly close the thread on client disconnect; maybe this should be something that continually tries to read data from the client (because maybe one day the client will actually send messages too?) and exits on exception. Example: https://github.com/jaspervdj/websockets/blob/master/example/server.lhs#L122